docs/user/dashboard/lens-percentage.asciidoc

```diff
@@ -0,0 +1,71 @@
+[[analyze-data-as-percentage-in-line]]
+== Advanced guide: Analyze data as percentage in Lens
+
+It is often useful to with with percentages within your data: from computing a single metric about performance goal, to representing the proportion of metrics over time.
+
+The aim of this guide is to provide an overview of representing percentages with several visualizations within the Lens editor: some of the content of this guide is inspired by this blog post based on TSVB, but applied to the Lens editor.
+
+In this guide we'll use the **flights** sample data sets to explore the different options we have to answer the following questions:
+
+* How are values distributed within my data?
+* What is the proportion of X within my data?
+* Is the distribution changing over time?
+
+=== Computing the proportion for the overall data
+
+The first step to understand the distribution is to have an overview for the overall data: this type of visualization is useful to compared multiple values to the whole dataset and understand their proportions.
+
+*Lens* offers different types of visualization in order to help with this type of question: *Pie* chart, *Donut* chart, *Treemap* and *Percentage* charts.
+
+In this example we'll investigate what are the most common reasons of flights delay within the dataset: the idea is to count all the available records, then group them by the type of delay recorded.
```

Comment:
I think this section is focused too much on _chart types_ and not enough on _aggregation types_ and _data types_. It seems like you've chosen to use the Top Values function for all the charts, but why? Can you explain to the user why you've chosen this function compared to the other functions? Can you explain to the user if there are cases when they would want to build a distribution using a date histogram? (I can think of a few)